packages/components/src/list/style.scss

```diff
@@ -7,11 +7,21 @@
 	display: flex;
 	align-items: center;
 	margin-bottom: 0;
+	text-decoration: none;
```

Comment:
I've duplicated some styles that were in list-item-inner up one level here. It doesn't affect existing styling so I think it's fairly low risk.